lib/src/Pos/Web/Api.hs

```diff
@@ -65,28 +65,27 @@ type HealthCheckApi =
 baseNodeApi :: Proxy BaseNodeApi
 baseNodeApi = Proxy
 
--- | GodTossing specific API.
-type GodTossingApi =
+-- | SSC specific API.
```

Comment:
These APIs should be merged. Previously we had «base» API and GodTossing-specific API. They were separated, because SSC wasn't equivalent to GodTossing. Now it is, so it doesn't make sense to keep them separated.